templates/20years/index.html

```diff
@@ -667,62 +667,99 @@ <h2>
           <div class="p-heading-icon">
             <div class="p-heading-icon__header">
               <img class="p-heading-icon__img"
-                   src="https://assets.ubuntu.com/v1/62a77319-@pleia2.png"
-                   alt="" />
+                   src="https://assets.ubuntu.com/v1/883761af-brandon%20lichtenwalner.jpeg"
+                   alt="brandon Lichtenwalner photo" />
               <p class="p-heading-icon__title">
-                <strong><small>Tweet by
-                @pleia2</small></strong>
+                <strong><small>LinkedIn post by Brandon Lichtenwalner</small></strong>
               </p>
             </div>
           </div>
           <p>
-            I haven't been on the <a href="https://x.com/hashtag/Ubuntu?src=hashtag_click">#Ubuntu</a> wagon the whole time, but I did do my first install on March 12th, 2005 (I know the exact date because I also signed up to the Ubuntu Forums that day), and requested a copy of the second release!
+            Can you believe Ubuntu has been around for 20 years!? I remember it changing the game when it first released, and it has been one of my go to distributions (for desktop and later server) ever since, even when it's not my daily driver.
           </p>
           <p>
-            Happy birthday 15th birthday! <a href="https://x.com/hashtag/Ubuntu15?src=hashtag_click">#Ubuntu15</a> 😁
+            Releases that stick out for me: 6.06 | 12.04 | 20.04
           </p>
           <div class="p-section--shallow">
-            {{ image(url="https://assets.ubuntu.com/v1/a5a694bc-post 8.png",
-                        alt="",
-                        width="1512",
-                        height="852",
-                        hi_def=True,
-                        loading="lazy") | safe
+            {{ image (
```

Comment:
```suggestion
            {{ image(
```
This is going to sound super annoying, but you have to remove the space and the run djlint. I know it seems silly (I pushed back on it), but it is what the team agreed on